.ci/pipeline/ci.yml

```diff
@@ -14,6 +14,26 @@
 # limitations under the License.
 #===============================================================================
 
+trigger:
+  branches:
+    include:
+    - master
+    - releases
+    - rls/*
+  paths:
+    exclude:
+    - requirements-doc.txt
```

Comment:
Only reqs? I think `doc` directory and other related files might be added

Reply:
added